src/routes/safe/components/Settings/Advanced/dataFetcher.ts

```diff
@@ -0,0 +1,43 @@
+import { List } from 'immutable'
+
+export const MODULES_TABLE_ADDRESS_ID = 'address'
+export const MODULES_TABLE_ACTIONS_ID = 'actions'
+
+export const getModuleData = (
+  modulesList: List<[string, string]>,
+): List<{ [MODULES_TABLE_ADDRESS_ID]: [string, string] }> => {
+  return modulesList.map((modules) => ({
+    [MODULES_TABLE_ADDRESS_ID]: modules,
+  }))
+}
+
+interface TableColumn {
+  align?: 'inherit' | 'left' | 'center' | 'right' | 'justify'
```

Comment:
maybe this interface can be extracted in a more generic place? it's being used in every table